mirrord/layer/tests/apps/listen_ports/src/main.rs

```diff
@@ -0,0 +1,20 @@
+#![warn(clippy::indexing_slicing)]
```

Comment:
Hmm, for some reason I was under the impression that we were not using full cargo projects for integration tests (at least when avoidable due to it having no dependencies).

Reply:
welp, I think it's worth having just for IDE support

Reply:
Yup, makes sense. I was writing my test samples in a separate project thinking this was the way :crying_cat_face: